server/plugin/subscriptions.go

```diff
@@ -284,66 +277,10 @@ func (p *Plugin) StoreSubscriptions(s *Subscriptions) error {
 	return nil
 }
 
-func (p *Plugin) GetExcludedNotificationRepos() ([]string, error) {
-	var subscriptions []string
-	value, appErr := p.API.KVGet(SubscribedRepoNotificationOff)
-	if appErr != nil {
-		return nil, errors.Wrap(appErr, "could not get subscriptions from KVStore")
-	}
-	if value == nil {
-		return []string{}, nil
-	}
-	err := json.NewDecoder(bytes.NewReader(value)).Decode(&subscriptions)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not properly decode subscriptions key")
-	}
-	return subscriptions, nil
+func GetExcludedNotificationRepos(s Subscription) []string {
+	return s.Flags.ExcludedRepos
```

Comment:
Since this function is only called once and it's a one-liner, it's sort of useless. 0/5 we should instead use `s.Flags.ExcludedRepos` in the one place we're calling this function

Reply:
Removed.

Reply:
Looks like it's still here

Reply:
Oh, my bad. I am removing it.